src/System.IO.Pipelines/ReadCursorOperations.cs

```diff
@@ -60,7 +60,7 @@ public static int Seek(ReadCursor begin, ReadCursor end, out ReadCursor result,
                 var segment = segmentPart.Segment;
                 var span = segment.Buffer.Span.Slice(segmentPart.Start, segmentPart.Length);
 
-                int index = span.IndexOfAny(byte0, byte1, byte2);
+                int index = System.SpanExtensions.IndexOfAny(span, byte0, byte1, byte2);
```

Comment:
These changes are unnecessary since we have IndexOfAny now, as of https://github.com/dotnet/corefx/pull/17357